javaharness/java/arcs/api/ArcsEnvironment.java

```diff
@@ -1,39 +1,11 @@
 package arcs.api;
 
-import java.util.List;
-
 /**
  * Encapsulates the execution environment for Arcs, which can be implemented, for example, as a
  * WebView, a separate Javascript interpreter, a WASM-ahead-of-time compiled or native code base.
  */
 public interface ArcsEnvironment {
```

Comment:
with everything that's been removed around this interface, is it even needed now?